### PR TITLE
local-up-cluster: use the newly built library always

### PR DESCRIPTION
**Pull request by dims**
Avoid using `cluster/kubectl.sh` which was a crutch, let's find the actual binary we built and use it.

#### What type of PR is this?
/kind cleanup

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: hack/local-up-cluster.sh [hunk 1]
```diff
@@ -1391,8 +1391,8 @@ if [[ "${ENABLE_DAEMON}" = false ]]; then
...
 
 if [[ "${KUBETEST_IN_DOCKER:-}" == "true" ]]; then
-  cluster/kubectl.sh config set-cluster local --server=https://localhost:6443 --certificate-authority=/var/run/kubernetes/server-ca.crt
-  cluster/kubectl.sh config set-credentials myself --client-key=/var/run/kubernetes/client-admin.key --client-certificate=/var/run/kubernetes/client-admin.crt
-  cluster/kubectl.sh config set-context local --cluster=local --user=myself
-  cluster/kubectl.sh config use-context local
+  ${KUBECTL} config set-cluster local --server=https://localhost:6443 --certificate-authority=/var/run/kubernetes/server-ca.crt
+  ${KUBECTL} config set-credentials myself --client-key=/var/run/kubernetes/client-admin.key --client-certificate=/var/run/kubernetes/client-admin.crt
+  ${KUBECTL} config set-context local --cluster=local --user=myself
+  ${KUBECTL} config use-context local
```
[the remark this answers is not in the document]
Comment (pohly) [reply hack/local-up-cluster.sh:1397]: Why does the PR comment mention `kubectl.sh` then?

[the remark this answers is not in the document]
Comment (pohly) [reply hack/local-up-cluster.sh:1397]: Because the removed code was using it...

Time for a break. Anyway:

/lgtm

**Comment by dims** [comment]
> Should this PR remove the set +x?

@pohly not yet, waiting for next kubekins-e2e images to hit the CI and cross check if the `docker stop` with `timeout` fixes it or not

/hold cancel

**Review by mtardy — COMMENTED** [review]
Thanks dims!! This will help me :)!

/lgtm

**Pull request merged**